syntax = "proto2";

package mivp_agent;

import "node_report.proto";

message AquaticusVehicle {
  required bool tagged = 1;
  required bool has_flag = 2;
  required NodeReport node_report = 3;
}